lib/her/model/base.rb

```diff
@@ -28,6 +28,17 @@ def [](attribute_name)
       def singularized_resource_name
         self.class.name.split('::').last.tableize.singularize
       end
+
+      # @private
+      def initialize_dup(other)
```

Comment:
can we go ahead and move this to `attributes.rb` below [initialize](https://github.com/remiprev/her/blob/master/lib/her/model/attributes.rb#L26)?